drift/app_config.py

```diff
@@ -1,8 +1,17 @@
 import os
 
+drift_app_version = "0.0.1"
```

Comment:
I tend to expect the version to be more in a setup.py file or VERSION file. This does depend on what the intended workflow is for bumping the version.